src/olympia/amo/mail.py

```diff
@@ -17,30 +17,39 @@ class DevEmailBackend(BaseEmailBackend):
     `settings.SEND_REAL_EMAIL` is disabled.
 
     BUT even if `settings.SEND_REAL_EMAIL` is disabled, if the targeted
-    email address is in the `settings.EMAIL_QA_ALLOW_LIST` list,
-    the email will be sent.
+    email address is in `dev_email` group users, the email will be sent.
     """
 
+    @staticmethod
+    def force_send_mail_group():
+        return 'Force Send Mail'
+
     def send_messages(self, messages):
         """Save a `FakeEmail` object viewable within the admin.
 
         If one of the target email addresses is in
-        `settings.EMAIL_QA_ALLOW_LIST`, it send a real email message.
+        `Group('force_send_mail_group')`, it send a real email message.
         """
         log.info('Sending dev mail messages.')
         qa_messages = []
         for msg in messages:
             FakeEmail.objects.create(message=msg.message().as_string())
-            qa_emails = set(msg.to).intersection(settings.EMAIL_QA_ALLOW_LIST)
-            if qa_emails:
-                if len(msg.to) != len(qa_emails):
-                    # We need to replace the recipients with the QA
-                    # emails only prior to send the message for real.
-                    # We don't want to send real emails to people if
-                    # they happen to also be in the recipients together
-                    # with white-listed emails
-                    msg.to = list(qa_emails)
-                qa_messages.append(msg)
+            try:
+                dev_email_group = Group.objects.get(name=self.force_send_mail_group())
```

Comment:
move this out of the for loop so we aren't `get`ting for each `msg`, and flatten it to a set once.

Reply:
Good catch.